project.clj

```diff
@@ -5,6 +5,7 @@
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.7.0"]
                  [org.clojure/clojurescript "1.7.170" :scope "provided"]
+                 [org.clojure/math.combinatorics "0.1.3" :scope "test"]
```

Comment:
Are you sure of this?  When I created a dummy project that has this version of clara as a dependency with no special profile or scope and uberjar'ed it, I was unable to find the combinatorics clj files inside the jar.